docs/api-reference/layers/editable-geojson-layer.md

```diff
@@ -75,20 +75,24 @@ The `mode` property dictates what type of edits the user can perform and how to
 
 * `drawLineString`: user can draw a `LineString` feature by clicking positions to add.
 
-  * If no feature is selected, clicking will create a new feature `Point` feature and select it (by passing its index as `updatedSelectedFeatureIndex`).
+  * If no feature is selected, clicking will create a new feature `Point` feature and select it (by passing its index as `updatedSelectedFeatureIndexes`).
 
   * If a `Point` feature is selected, clicking will convert it to a `LineString` and add the clicked position to it.
 
   * If a `LineString` feature is selected, clicking will add the clicked position to it.
 
+  * If multiple features are selected, the user will be prevented from drawing
+
 * `drawPolygon`: user can draw a new `Polygon` feature by clicking positions to add then closing the polygon.
 
-  * If no feature is selected, clicking will create a new feature `Point` feature and select it (by passing its index as `updatedSelectedFeatureIndex`).
+  * If no feature is selected, clicking will create a new feature `Point` feature and select it (by passing its index as `updatedSelectedFeatureIndexes`).
 
   * If a `Point` feature is selected, clicking will convert it to a `LineString` and add the clicked position to it.
 
   * If a `LineString` feature is selected, clicking will add the clicked position to it. If that clicked position is the first position, the feature will be converted to a `Polygon` feature.
 
+  * If multiple features are selected, the user will be prevented from drawing
+
 #### `onEdit` (Function, optional)
```

Comment:
Oops, I never documented the `selectedFeatureIndex` prop. This is a good time to fix that and document the `selectedFeatureIndexes` prop.